igvm/hypervisor.py

```diff
@@ -65,17 +65,31 @@ def __init__(self, dataset_obj):
         self._mount_path = {}
         self._storage_type = None
 
-    def get_storage_pool(self) -> virStoragePool:
+    def get_active_storage_pools(self):
+        # Return all active storage pools
+        return self.conn().listAllStoragePools(2)
```

Comment:
```suggestion
        # The 2 used as argument is the value of the VIR_CONNECT_LIST_STORAGE_POOLS_ACTIVE flag.
        return self.conn().listAllStoragePools(flags=2)
```

I think the comment change and specifying the keyword flags make it easier to understand.